registration/models.py

```diff
@@ -91,9 +91,6 @@ def clean(self):
             raise ValidationError(
                 _("Provide a passport country of origin in the \
                    mom_passport_origin field."))
-        if self.mom_id_type == 'none' and self.mom_dob is None:
```

Comment:
We should leave this in but add an extra qualifier, we still care about this for clinic etc: `if self.authority = 'clinic' and ...`